src/modules/auth/models/user-model.js

```diff
@@ -13,6 +13,9 @@ const schema = mongoose.Schema({
     is_verified:{
         type:Boolean,
         default:false,
+    },
+    newPassword:{
```

Comment:
bro, you need to update the password instead of writing a new password in new field. also, i said you to write key's with _ format